shared/rake/test.rake

```diff
@@ -1,5 +1,16 @@
 require 'rake/testtask'
-task :test do
+require 'cdo/rake_utils'
+
+task :prepare_dbs do
+  with_rack_env(:test) do
+    Dir.chdir(pegasus_dir) do
+      puts 'Migrating pegasus test database...'
+      RakeUtils.rake 'db:ensure_created', 'db:migrate', 'seed:migrate'
```

Comment:
I wonder if invoking these Rake tasks from within the existing process (using `Rake::Task['db:ensure_created'].invoke` rather than `RakeUtils.rake`) would make this preparation run any quicker.

Reply:
The replacement you suggested here doesn't work out of the box, because it can't find the `db:ensure_created` task.  I think that's because we're in `shared/rake/test.rake` but we're trying to run tasks in `pegasus/rake/db.rake` (hence the directory change).  Do you know if there's a clean way to make those pegasus tasks available without exposing them all as part of this rakefile?